software/materialious

```diff
@@ -0,0 +1,13 @@
+name: Materialious
+website_url: https://materialio.us
+description: Modern material design for Invidious. 
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+tags:
+  - Media Streaming - Video Streaming
+source_code_url: https://github.com/WardPearce/Materialious
+stargazers_count: 222
+updated_at: '2024-04-11'
+archived: false
```

Comment:
```suggestion

```
Feel free to omit this line, it'll be added in automatically.

Reply:
👍